conslogging/conslogging.go

```diff
@@ -221,3 +232,19 @@ func (cl ConsoleLogger) color(c *color.Color) *color.Color {
 	}
 	return noColor
 }
+
+func (cl ConsoleLogger) prettyPrefix() string {
+	if cl.prefixPadding == NoPadding {
+		return cl.prefix
+	}
+
+	formatString := fmt.Sprintf("%%%vv", cl.prefixPadding)
+	substringStart := len(cl.prefix) - cl.prefixPadding
+
+	clampedPadding := substringStart
+	if clampedPadding < 0 {
+		clampedPadding = 0
+	}
+
+	return fmt.Sprintf(formatString, cl.prefix[clampedPadding:])
+}
```

Comment:
I am torn here. I do not like calculating this on each log right now... Should I move this up into the main struct alongside regular prefix, and calculate it once? I'm probably going to do that.